From 986a1786fbbc55651d21209c1e51fb4b74deb616 Mon Sep 17 00:00:00 2001
From: "Angelo G. Del Regno" <kholk11@gmail.com>
Date: Wed, 27 Mar 2019 18:16:45 +0100
Subject: [PATCH 125/175] clk: dispcc-sdm845: Do not gate by default DSI,
 pixel, byte clocks

Disable clock gating by default for the DSI, pixel and byte:
now the SDE driver is responsible of selectively enabling
clock gating for each of these clocks by setting the single
corresponding bit.
---
 drivers/clk/qcom/dispcc-sdm845.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/clk/qcom/dispcc-sdm845.c b/drivers/clk/qcom/dispcc-sdm845.c
index d37113e8246a..8ec507456805 100644
--- a/drivers/clk/qcom/dispcc-sdm845.c
+++ b/drivers/clk/qcom/dispcc-sdm845.c
@@ -1134,7 +1134,7 @@ static int disp_cc_sdm845_probe(struct platform_device *pdev)
 	clk_fabia_pll_configure(&disp_cc_pll0, regmap, &disp_cc_pll0_config);
 
 	/* Enable clock gating for DSI and MDP clocks */
-	regmap_update_bits(regmap, DISP_CC_MISC_CMD, 0x7f0, 0x7f0);
+	regmap_update_bits(regmap, DISP_CC_MISC_CMD, 0x10, 0x10);
 
 	ret = disp_cc_sdm845_fixup(pdev, regmap);
 	if (ret)
-- 
2.48.1

